setup.cfg

```diff
@@ -0,0 +1,44 @@
+[metadata]
+name = enoslib
+version = 1.2.1
+keywords = evaluation, reproducible research, Grid5000
+license_file = LICENSE.txt
+url = https://github.com/beyondtheclouds/enoslib
+author = Matthieu Simonin
+author_email = matthieu.simonin@inria.fr
+long_description = file: README.md
+classifier =
+    Development Status :: 4 - Beta
+    Intended Audience :: System Administrators
+    Intended Audience :: Science/Research
+    License :: OSI Approved :: GNU General Public License v3 (GPLv3)
+    Operating System :: POSIX :: Linux
+    Programming Language :: Python :: 2.7
+
+[options]
+include_package_data = True
+packages = find:
+setup_requires =
+    setuptools>=39.0.1
+    wheel>=0.31.0
+install_requires =
+    ansible>=2.5.0,<2.6.0
+    jsonschema >= 2.6.0, < 2.7
+    execo >= 2.6.2, < 2.7
+    requests>=2.18.0, <2.19
+    python-vagrant>=0.5.15
+    netaddr>=0.7,<0.8    
+
+[options.extras_require]
```

Comment:
👍 
Maybe need to update the doc at some point.

Reply:
done ;)